File: sudoku-gateway/src/main/java/com/zumin/sudoku/gateway/SudokuGatewayApplication.java
package com.zumin.sudoku.gateway;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

@EnableDiscoveryClient
@SpringBootApplication
public class SudokuGatewayApplication {

  public static void main(String[] args) {
    SpringApplication.run(SudokuGatewayApplication.class, args);
  }

}

File: sudoku-gateway/src/main/java/com/zumin/sudoku/gateway/SudokuGatewayApplication.kt
package com.zumin.sudoku.gateway

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@EnableDiscoveryClient
@SpringBootApplication
class SudokuGatewayApplication

fun main(args: Array<String>) {
  runApplication<SudokuGatewayApplication>(*args)
}